webpack/components/AnsibleHostDetail/index.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import AnsibleHostDetail from './AnsibleHostDetail';
+
+const WrappedAnsibleHostDetail = props => <AnsibleHostDetail {...props} />;
```

Comment:
You can just do 
```js
export {default as AnsibleHostDetail} from './AnsibleHostDetail';
```
though I understand this is just an initial step, this file will be changed